assets.go

```diff
@@ -175,6 +175,16 @@ func LoadShader(vertSrc, fragSrc string) *gl.Program {
 	vertShader := Gl.CreateShader(Gl.VERTEX_SHADER)
 	Gl.ShaderSource(vertShader, vertSrc)
 	Gl.CompileShader(vertShader)
+	var success int32
+	Gl.GetShaderiv(vertShader, Gl.COMPILE_STATUS, &success)
+	if success != int32(Gl.TRUE) {
+		var maxLength int32
+		Gl.GetShaderiv(vertShader, Gl.INFO_LOG_LENGTH, &maxLength)
+		log.Println("Error during vertex shader compilation")
+		errorLog := make([]byte, maxLength)
+		Gl.GetShaderInfoLog(vertShader, maxLength, &maxLength, (*uint8)(Gl.Ptr(errorLog)))
+		log.Print(string(errorLog))
```

Comment:
Yeah, but if the string already contains a newline, `log.Println` will result in double newlines (since `log.Println` uses `fmt.Sprintln`,  which uses `fmt.doPrint(...)` which simply appends a newline. 

So if we were to use `log.Println`, it would result in a double newline in the logs. That's why we won't be using `Println` :smile: .

Thanks for doing the research though.